Update README.md files in json-to-jms and upload-to-ftp guides (#372)
wso2/ballerina-integrator: docs/content/src/learn/guides/messaging/sending-json-data-to-a-jms-queue/README.md

<!-- INCLUDE_CODE: src/jsontojms/main.bal -->

Same as JMS API, create `jms:Connection` then `jms:Session` and finally create `jms:Destination` and `jms:MessageProducer`. In the HTTP resource, build `jms:TextMessage` getting the payload as a `string`. Next, send the message to the sales queue in ActiveMQ. Once the message is sent, verify if the return result is an error. If it is an error, build `http:Response` as the JSON payload. If it is a success, do the same. As the last step, respond to the caller with build `http:Response`.

## Run the integration

> Before building the module, we have to copy the necessary ActiveMQ dependencies into the project. There are three jar files listed down below. These jar files you can find in the lib folder of ActiveMQ distribution.

Reviewer: ```suggestion
Before building the module, we have to copy the necessary ActiveMQ dependencies into the project. There are three jar files listed down below. These .jar files can be found in the `lib` folder of the ActiveMQ distribution.
```

http://localhost:8080/sales/orders
```

> You will see the following response in a successful invocation. Also, the published message should be able visible in the sales queue when you log into the ActiveMQ web console.  

Reviewer: ```suggestion
You will see the following response in a successful invocation. Also, the published message should be visible in the sales queue when you log into the ActiveMQ web console.  
```
